scripts/run-local-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+ACCOUNT_ID=XXX
+ACCESS_TOKEN=XXX
+USERNAME=betademo
+PASSWORD=nottobes
```

Comment:
We decided to move those credentials to `BeforeSuite()` and generate them so they are not hardcoded here.